R/pkg/tests/fulltests/test_eager_execution.R

```diff
@@ -0,0 +1,52 @@
+#
```

Comment:
one thing to note, since test runs alphebatically, this test will run before sparkSQL - I think we should rename this to test_sparkSQL_eager.R to ensure it runs after

Reply:
Done.